zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/CuratorDiscoverySpec.java

```diff
@@ -42,7 +46,12 @@ public String path() {
 
     @Override
     public Endpoint decode(byte[] data) {
-        return converter.apply(CuratorXNodeValueCodec.INSTANCE.decode(data));
+        final ServiceInstance<?> decodedInstance = CuratorXNodeValueCodec.INSTANCE.decode(data);
+        final Endpoint endpoint = converter.apply(decodedInstance);
+        if (endpoint == null) {
+            logger.debug("Returned null endpoint from {}.", decodedInstance);
```

Comment:
- Should we log at WARN level instead of DEBUG?
- How about `The endpoint converter returned null from {}.` ?